app/gateway/changelog.md

```diff
@@ -9,6 +9,431 @@ Changelog for supported Kong Gateway versions.
+* Added support for debugging with EmmyLuaDebugger. This feature is in
+tech preview and is not officially supported by Kong Inc.
+ [#12899](https://github.com/Kong/kong/issues/12899)
+* **Analytics**: 
+  * Added the `latencies.receive_ms` and `websocket` fields.
+  * Removed received time and latency from `latencies.kong_gateway_ms`.
+  * Added the `sse` boolean field to the payload, which is set to `true` for Server-Sent Event requests and responses.
+
+#### Kong Manager
+* Kong Manager now supports creating and editing expressions routes using interactive in-browser editor with syntax highlighting and 
+autocompletion features for Kong's Expressions language.
+ [#217](https://github.com/Kong/kong-manager/issues/217)
+* Kong Manager now groups parameters to provide a better user experience while configuring plugins.
+* When authenticating Kong Manager with IDPs (for example, OIDC or LDAP), the source of an RBAC role is now stored in its `role_source` field. This enables the existing roles with a source of `idp` to be removed upon new logins after IDP role mapping has changed. This also allows users to switch a role's source between `local` and `idp`.
+
+#### PDK
+* Added the `latencies.receive` property to the log serializer.
+ [#12730](https://github.com/Kong/kong/issues/12730)
+
+#### Plugins
+
+**New plugins**:
+* [**AI Rate Limiting Advanced**](/hub/kong-inc/ai-rate-limiting-advanced/) (`ai-rate-limiting-advanced`): 
+This plugin lets you implement a rate limit by an AI provider.
+* [**AI Azure Content Safety**](/hub/kong-inc/ai-azure-content-safety/) (`ai-azure-content-safety`): 
+Lets you enforce introspection of all AI Proxy requests with the Azure Content Safety service.
+The plugin enables configurable thresholds for the different moderation categories, 
+and reports audit results into the Kong log serializer for reporting purposes.
+
+**Updates to existing plugins:**
+
+* [**AI Proxy**](/hub/kong-inc/ai-proxy/) (`ai-proxy`)
+  * AI Proxy now reads most prompt tuning parameters from the client, 
+  while the plugin config parameters under `model_options` are now just defaults.
+  This fixes support for using the respective provider's native SDK.
+ [#12903](https://github.com/Kong/kong/issues/12903)
+  * AI Proxy now has a [`preserve` option for `route_type`](/hub/kong-inc/ai-proxy/configuration/#config-route_type), 
+  where the requests and responses are passed directly to the upstream LLM. This enables compatibility with any
+  models and SDKs that may be used when calling the AI services.
+ [#12903](https://github.com/Kong/kong/issues/12903)
+  * Added support for [streaming event-by-event responses](/hub/kong-inc/ai-proxy/how-to/streaming/) back to the client on supported providers.
+ [#12792](https://github.com/Kong/kong/issues/12792)
+  * **Enterprise-only feature**: Added support for [Managed Identity authentication](/hub/kong-inc/ai-proxy/how-to/cloud-provider-authentication/) 
+  when using the Azure provider with AI Proxy.
```

Comment:
I don't think I added this one to the AI Proxy plugin changelog, should we add this to that plugin changelog as well?